content/en/real_user_monitoring/guide/upload-javascript-source-maps.md

```diff
@@ -117,31 +117,36 @@ The best way to upload source maps is to add an extra step in your CI pipeline a
 
 To minimize overhead on your CI's performance, the CLI is optimized to upload as many source maps as you need in a short amount of time (typically a few seconds).
 
-By running the command against the example `dist` directory, Datadog expects your server or CDN to deliver the Javascript files at `https://hostname.com/static/js/javascript.364758.min.js` and `https://hostname.com/static/js/subdirectory/javascript.464388.min.js`. 
+The `--service` and `--release-version` parameters must match the `service` and `version` tags on your RUM events. For more information on how to setup these tags, refer to the [Browser SDK initialization documentation][2]. The sourcemaps that have been uploaded will then be used to de-obfuscate errors collected by the RUM SDK.
 
-The RUM SDK instantly collects errors that occur in user sessions. When a given error originates in a file that was downloaded from one of these URLs and is tagged with `version:v35.2395005` and `service:my-service`, the related source map is used to de-obfuscate the stack trace (in this example, the `javascript.464388.js.map` file).
+By running the command against the example `dist` directory, Datadog expects your server or CDN to deliver the Javascript files at `https://hostname.com/static/js/javascript.364758.min.js` and `https://hostname.com/static/js/subdirectory/javascript.464388.min.js`.
 
 **Note**: Only source maps with the `.js.map` extension work to correctly unminify stack traces in Error Tracking. Source maps with other extensions such as `.mjs.map` are accepted but do not unminify stack traces.
 
 <div class="alert alert-info">If you are serving the same JavaScript source files from different subdomains, upload the related source map once and make it work for multiple subdomains by using the absolute prefix path instead of the full URL. For example, specify <code>/static/js</code> instead of <code>https://hostname.com/static/js</code>).</div>
 
-If you have successfully uploaded source maps but they do not appear in the RUM dashboard, see [RUM Browser Troubleshooting][2].
+### Link stack frames to your source code
+
+If you run `datadog-ci sourcemaps upload` within a Git repository, it will automatically collect metadata about the repository, including the repository URL, current commit hash and the list of file paths in the repository that relate to your source maps. For more details about git metadata collection, refer to [datadog-ci documentation][3].
```

Comment:
```suggestion
Within a Git working directory, run `datadog-ci sourcemaps upload` to collect repository metadata and upload it to Datadog. The `datadog-ci` command collects the repository URL, the current commit hash, and the list of file paths in the repository that relate to your source maps. For more details about Git metadata collection, refer to [datadog-ci documentation][3].
```

Reply:
To clarify, here we're talking about the same `datadog-ci` command than above - it only needs to be ran once. With this phrasing, I'm afraid people could think that they're required to run it twice. What do you think about
```suggestion
Run `datadog-ci sourcemaps upload` within a Git working directory to collect repository metadata and upload it to Datadog. The `datadog-ci` command collects the repository URL, the current commit hash and the list of file paths in the repository that relate to your source maps. For more details about Git metadata collection, refer to the [datadog-ci documentation][3].
``` 
?

Reply:
Sure, I like your phrasing too.

My main goal was to break the original sentence into 2 smaller ones: one sentence to tell people what the command does, and one sentence to elaborate on which metadata is uploaded. We prefer shorter sentences in the documentation because they are easier for users to understand, and for our translators to translate.

Feel free to put the "If you run" phrasing back into the doc. I didn't realize I had changed the meaning a bit by taking it out. I thought it wasn't necessary and would make the doc clearer to remove it, but I now see that you may have had it in there to show people they only have to run the command once.

Reply:
Done !